tls/extensions/s2n_client_signature_algorithms_cert.c

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include <sys/param.h>
+#include <stdint.h>
+
+#include "tls/extensions/s2n_client_signature_algorithms_cert.h"
+#include "tls/s2n_connection.h"
+#include "tls/s2n_tls.h"
+#include "tls/s2n_tls_parameters.h"
+#include "tls/s2n_signature_algorithms.h"
+
+#include "stuffer/s2n_stuffer.h"
+#include "utils/s2n_safety.h"
+
+static int s2n_client_signature_algorithms_cert_recv(struct s2n_connection *conn, struct s2n_stuffer *extension);
```

Comment:
Yeah I sort of went back and forth about splitting it; I thought it might be more readable if I did split it because then we wouldn't be reusing code. But as long as that doesn't mess with how we do things in s2n, I would be more inclined to only have one s2n_signature_algorithms_cert file.